frontend/src/static/js/components/styles/NavigationMenuList.scss

```
@@ -86,7 +85,7 @@
 	.menu-item-icon {
 		margin-right: 24px;
 
-		color: #888;
+		color: var(--text-muted);
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)